src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/Language/Syntax/GreenNode.cs

```diff
@@ -251,31 +251,26 @@ public virtual string ToFullString()
         return builder.ToString();
     }
 
-    public virtual void WriteTo(TextWriter writer)
-    {
-        WriteTo(writer, includeLeadingTrivia: true, includeTrailingTrivia: true);
-    }
-
-    protected internal void WriteTo(TextWriter writer, bool includeLeadingTrivia, bool includeTrailingTrivia)
+    public void WriteTo(TextWriter writer)
     {
         // Use an actual Stack so we can write out deeply recursive structures without overflowing.
-        using var stack = new PooledArrayBuilder<StackEntry>();
+        using var stack = new PooledArrayBuilder<GreenNode>();
 
-        stack.Push(new(Node: this, includeLeadingTrivia, includeTrailingTrivia));
+        stack.Push(this);
 
         // Separated out stack processing logic so that it does not unintentionally refer to
         // "this", "leading" or "trailing.
         ProcessStack(writer, ref stack.AsRef());
```

Comment:
I don't totally follow this comment, but given that we've now removed the leading and trailing parts, can it just be inlined?